packages/hebao_v1/contracts/modules/security/RecoveryModule.sol

```diff
@@ -63,7 +64,14 @@ contract RecoveryModule is SecurityModule
         require(newOwner != address(0), "ZERO_ADDRESS");
 
         w.setOwner(newOwner);
-        emit Recovered(wallet, oldOwner, newOwner);
+
+        SecurityStore securityStore = controller.securityStore();
+        bool newOwnerIsGuardian = securityStore.isGuardian(wallet, newOwner);
```

Comment:
newOwner isGuardian or isPendingAddition. should add a new method in securityStore: isGuardianOrPendingAddition.